src/main/java/com/hivemq/client/mqtt/mqtt3/message/publish/Mqtt3Publish.java

```diff
@@ -77,6 +77,12 @@ public interface Mqtt3Publish extends Mqtt3Message {
      */
     boolean isRetain();
 
+    /**
+     * @return the optional local correlation data of this Publish message. This data is never propagated and kept
+     * locally for correlation.
+     */
+    Object localCorrelationData();
```

Comment:
```suggestion
    Object getLocalContext();
```
Correlation data is used to correlate different publishes (sending publish A and receiving publish B which contains the same correlation data).
Context is imho more appropriate. Context is also used in reactor terminology.

Reply:
Changed the name in all places.